bundles.md

```diff
@@ -26,6 +26,7 @@ Provide detailed information about a specific bundle. A sample JSON response doc
         }
       ]
   },
+  "primaryBitstreamUUID": "ac49f361-4ffd-47a4-8eb2-e6c73c3f3e76",
```

Comment:
This can likely be removed as the primarybitstream is available in the `_links` (no need to duplicate it here).  I think this was leftover from the old design for this endpoint.

Reply:
I've removed the property from the contract